### ISSUE TITLE
Note on internationalization needs for non-visible text?

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#4658]
In https://github.com/w3c/wcag/pull/4482#issuecomment-3349249256 it was pointed out that non-visible text is more easily overlooked during translation.

> Localization reminder: It may be helpful to note that any aria-label text should be localized, with a simple pointer to W3C i18n guidance so that readers don’t overlook translation and localization considerations.

Any use of the `alt` attribute will likewise create a scenario where it is more likely for language to be overlooked for translation, since it is not exposed visually by user agents by default. The same consideration applies to a lesser to degree to values of some attributes like `title`which are only exposed after specific user actions. Should the working group compose a note be incorporated into relevant techniques?

**Comment by TestPartners** [comment]
Whilst localisation is obviously important, I don't think such notes should be included in the specifications for individual ARIA attributes. Localisation applies to many types of "hidden" content, such as those created using CSS content, DOM scripting or asynchronous loading. It applies to captions, transcripts, `<title>` elements and maybe some `<meta>` tags.

In my view, specifications for WCAG, ARIA etc. should just contain one comprehensive section that attempts to cover all the relevant localisation requirements rather than adding notes throughout the specification.

**Comment by patrickhlauke** [comment]
there's a point where we can't add notes everywhere for everything

**Comment by mbgower** [comment]
> rather than adding notes throughout the specification.

Please note that this issue only mentions techniques. There is no proposal to make a change in the specification itself.

While I also have concerns with endemic appearances of such wording, it is clearly the case that the information provided through aria-label and alt text is more prone to being overlooked in translation, and so has a measurable impact on those most reliant on this content who do not happen to speak the source page's language.

**Comment by adampage** [comment]
If we do end up adding a note anywhere, the ARIA spec has a tidy “[Translatable Attributes](https://w3c.github.io/aria/#translatable-attributes)” section for potential cross-referencing, including lesser-used attributes like `aria-braillelabel`, `aria-colindextext`, etc.

The [HTML spec also has its own section](https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes) to cover host language attributes like `alt`, `title`, `th[abbr]`, etc.

**Comment by mbgower** [comment]
Perhaps we could consider adding something at the guideline level, such as at:

- https://www.w3.org/WAI/WCAG21/Understanding/text-alternatives
- https://www.w3.org/WAI/WCAG21/Understanding/readable
- https://www.w3.org/WAI/WCAG21/Understanding/compatible
